resources/containers/field-modal.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from "react";
+import Modal from "react-modal";
+import styles from "./styles/field-modal.module.css";
+
+interface FieldModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onSave: (data: any) => void;
+  fieldType: string;
+}
+
+const FieldModal: React.FC<FieldModalProps> = ({
```

Comment:
This is not a resource  xD is really coupled to the template needs, is a `partial` on the templates module

Reply:
I don't understand. Should I remove it? 

I felt this made the templates detail page less cluttered

Reply:
This is a correct subcomponent nd is good you took it out.

Just. need to be moved to `templates/components` , as the logic is coupled to the templates

`resource` folder has generic logics that can be used in any component